src/services/projects.js

```diff
@@ -39,7 +39,8 @@ angular.module('openshiftCommonServices')
       };
 
       return {
-        get: function(projectName) {
+        get: function(projectName, opts) {
+          opts = opts || {};
```

Comment:
nit: you can remove this line since you use `_.get` below, which handles undefined values

...or you can leave this line and check the property directly below without `_.get`

```js
if ((e.status === 403 || e.status === 404) && opts.skipErrorNotFound) {
```

Reply:
fixed, removed line